platform/darwin/src/MGLLocationManager.h

```diff
@@ -0,0 +1,181 @@
+#import <Foundation/Foundation.h>
+#import <CoreLocation/CoreLocation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol MGLLocationManagerDelegate;
+
+/**
+ The `MGLLocationManager` protocol defines a set of methods that a class must
+ implement in order to serve as the location manager of an `MGLMapView`. A location
+ manager is responsible for notifying the map view about location-related events,
+ such as a change in the user’s location. This protocol is similar to the
+ Core Location framework’s `CLLocationManager` class, but your implementation
+ does not need to be based on `CLLocationManager`.
+ 
+ */
+@protocol MGLLocationManager <NSObject>
+
+@optional
+
+#pragma mark Configuring Location Update Precision
+
+/**
+ Specifies the minimum distance (measured in meters) a device must move horizontally
+ before a location update is generated.
+ 
+ @see `[CLLocationManager distanceFilter]`
+ 
+ The default value of this property is `kCLDistanceFilterNone` when `MGLMapView` uses its
+ default location manager.
+ */
+@property(nonatomic, assign) CLLocationDistance distanceFilter;
+
+/**
+ Specifies the accuracy of the location data.
+ 
+ @see `[CLLocationManager desiredAccuracy]`
```

Comment:
Let’s put these `@see`s at the end of the documentation blocks, just after the `@note`s — otherwise it breaks up the flow in jazzy a bit:

<img width="900" alt="screen shot 2018-07-20 at 2 43 52 pm" src="https://user-images.githubusercontent.com/1198851/43019619-96e9b514-8c2b-11e8-8aef-fecb23f1a35e.png">